run/events-pubsub/requirements.txt

```diff
@@ -0,0 +1,32 @@
+astroid==2.4.2
```

Comment:
This is pretty long `requirements.txt` file. Are all these packages needed to run the sample?

If some are for tests only, please create a separate file `requirements-test.txt`

Reply:
Understood, and yeah I'm sure many of the packages aren't needed so I'll organize accordingly!

Reply:
iirc your CI tests wanted pytests in the requirements.txt. Should I have pytest installed separately in a requirements-test.txt?